executor/src/test/java/org/k8loud/executor/action/kubernetes/http/addResource.http

```diff
@@ -8,10 +8,51 @@ Content-Type: application/json
   "params": {
     "params": {
       "namespace": "default",
-      "resourceDescription": ""
+      "resourceDescription": "            apiVersion: v1\r\n            kind: Pod\r\n            metadata:\r\n              name: nginx\r\n            spec:\r\n              containers:\r\n              - name: nginx\r\n                image: nginx:1.14.2\r\n                ports:\r\n                - containerPort: 80"
     }
   }
 }
 
-### sending YAML in JSON is a pain, all whitechars need to be escaped
-### not bothering with solving this issue since we're getting rid of REST
+### add config map
+POST localhost:8080/execute
+Content-Type: application/json
+
+{
+  "collectionName": "kubernetes",
+  "actionName": "AddResourceAction",
+  "params": {
+    "params": {
+      "namespace": "default",
+      "resourceDescription": "            kind: ConfigMap\r\n            metadata:\r\n              creationTimestamp: 2022-02-18T18:52:05Z\r\n              name: game-config\r\n              namespace: default\r\n              uid: b4952dc3-d670-11e5-8cd0-68f728db1985\r\n            data:\r\n              game.properties: |\r\n                enemies=aliens\r\n                lives=3\r\n                enemies.cheat=true\r\n                enemies.cheat.level=noGoodRotten\r\n                secret.code.passphrase=UUDDLRLRBABAS\r\n                secret.code.allowed=true\r\n                secret.code.lives=30\r\n              ui.properties: |\r\n                color.good=purple\r\n                color.bad=yellow\r\n                allow.textmode=true\r\n                how.nice.to.look=fairlyNice"
```

Comment:
for now its ok, but can be json format. create story 😄

Reply:
https://github.com/k8loud/themis-executor/issues/77